.devbots/needs-triage.yml

```diff
@@ -0,0 +1,10 @@
+# adds a triage 5 minutes after creation.
+# triagers should replace this label with any appropriate labels, then, the cc-bot will cc interested parties.
+enabled: true
+label: "needs triage"
+delay: 5
+# issues only have assignees, milestones, or these labels when a core contributor created them.
+# no need to add the triage label in that case
+skip_when_has_assignee: true
+skip_when_has_milestone: true
+skip_when_has_labels: ["priority: 0", "priority: 2", "priority: 3"]
```

Comment:
can't we add any label here that is not part of the templates? (like all the module-specific labels)

Reply:
We can add wildcard `*` but the issue templates add `bug`, `feature`, `refactor` labels by default so we cannot use it

Reply:
Yes, but all the others we can, right? So whenever something except `bug`, `feature`, `refactor`, `docs` is present.

Reply:
Alternatively we can either list here all other labels
Or remove the default additions from the issue templates and have triagers label them

Reply:
Going with the 2nd option as it would be annoying to open a PR every time a label is added